packages/core/src/binder.ts

```diff
@@ -622,26 +622,72 @@ export function createOutputBinder(options: BinderOptions = {}): Binder {
   }
 
   function instantiateSymbolInto(source: OutputSymbol, target: OutputSymbol) {
-    if (~source.flags & OutputSymbolFlags.InstanceMemberContainer) {
-      throw new Error("Can only instantiate symbols with instance members");
-    }
-
+    // Ensure both member scopes exist on the target
     addInstanceMembersToSymbol(target);
+    addStaticMembersToSymbol(target);
```

Comment:
Instantiation should only ever result in static member symbols. Instantiated member symbols on the source become static symbols on the target. I suspect you won't be able to reference instantiated member symbols with this implementation?